datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -269,15 +290,30 @@ private static void initS3() {
             return;
         }
 
-        amazonS3 = AmazonS3ClientBuilder.standard()
-        .withCredentials(new AWSStaticCredentialsProvider(
-                new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
-        )
-        .withRegion(amazonRegion)
-        .build();
+        AmazonS3ClientBuilder amazonS3ClientBuilder = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(
+                        new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
+                );
+
+        String[] buckets = amazonBucket.split(",");
+        if (buckets.length > 1) {
+            log.info("Using {} buckets:{}", buckets.length, amazonBucket);
+        }
+
+        if (StringUtils.isNullOrEmpty(amazonUrl)) {
```

Comment:
Maybe for multi-bucket we can have separate S3-storage client?
so we get initMultiS3(), etc. functions

Reply:
amazonUrl is for ceph